src/services/aws/aws.ts

```diff
@@ -199,6 +202,18 @@ export const aws = createApi({
         };
       },
     }),
+    donations: builder.query<
```

Comment:
endpoint definitions has been moved from `services/apes` to `services/ap` as lambda also exist in AP